px/lg/values.yaml

```diff
@@ -24,3 +24,12 @@ looking_glass:
     proxy_port: 5005
     subdomain: pxadmin
     privileged: true
+
+resources:
+  lg:
+    requests:
+      cpu: "25m"
+      memory: "250Mi"
```

Comment:
The VPA upperbound recommendation is actually more in that range. I would use that as a limit. `requests` is what is allocated on container start at the machine. I don't think we need to allocate that much.